dev/source/docs/skyviper.rst

```diff
@@ -40,15 +40,15 @@ Otherwise:
 
 ::
 
-   git clone https://github.com/SkyRocketToys/ardupilot
+   git clone --recurse-submodules https://github.com/SkyRocketToys/ardupilot
 
 Once you have the repository:
 
 ::
 
    cd ardupilot
    git checkout skyviper-stable # or skyviper-latest
-   git submodule update --init --recursive
+   git submodule update
```

Comment:
switching branches here - need the init


... also, these steps are probably broken; better to run master?

Reply:
I'm not familiar with skyviper, but that branch is 30 commits in front of their master. I assume for good reason. 
https://github.com/SkyRocketToys/ardupilot/tree/skyviper-stable

Reply:
Added the init in 13d09cd0b61855ed5583280067b66d5fdd8e7b47